packages/connect-form/src/components/connect-form-actions.tsx

```diff
@@ -11,37 +11,76 @@ import {
   ConnectionFormError,
   ConnectionFormWarning,
 } from '../utils/validation';
+import type { ConnectionInfo } from 'mongodb-data-service';
 
 const formActionStyles = css({
   borderTop: `1px solid ${uiColors.gray.light2}`,
   paddingLeft: spacing[4],
   paddingRight: spacing[4],
 });
 
+const formActionButtonStyles = css({
+  display: 'flex',
+  justifyContent: 'flex-end',
+  '& div:first-child': {
```

Comment:
with flex you don't need to do this, you can specify a gap: https://css-tricks.com/almanac/properties/g/gap/

I'm not an expert and maybe there are other properties and tricks, but definitely adding a margin with `div:first-child` should not be needed.

Note that even keeping this solution, the selector: `'& div:first-child':` is technically a hack and not really a clean solution, maybe a bit better with a "direct child" selector `> ..` but you should really use a class to clarify the intention and make sure that future changes won't mess the style: `.action-button:first-child { ... }` and `<div className={cx('action-button', actionButtonStyles)}>`.

Take a look at the `gap` property though, we can use flex and avoid these weird selectors entirely